### DISCUSSION TITLE
GEFS static files

### DISCUSSION THREAD

**Discussion by MarcusGDaniels** [NOAA-EMC/global-workflow#1691] [General]
Hi,

Is there a git, S3, ftp, or other place to acquire the "fix" files for GEFS?  Trees like this from the link_gefs.sh script of GEFS?

emc.enspara/common/git/fv3gefs/fix_20200927

Thanks,

Marcus

**Comment by aerorahul** [comment]
I am not sure if this will help, but the current GEFS v12.3.8 fix files are available from the operational site for [GEFS code](https://www.nco.ncep.noaa.gov/pmb/codes/nwprod/gefs.v12.3.8/fix/)
